sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/compatibility/CompositeCompatibilityVerifier.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.compatibility;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+class CompositeCompatibilityVerifier {
+    private static final Log LOGGER = LogFactory.getLog(CompositeCompatibilityVerifier.class);
```

Comment:
why use commons.longging here?

Reply:
it's better to use org.slf4j.LoggerFactory here, we have 70+ files uses this  in spring projects.